jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ResourceHandler.java

```diff
@@ -89,7 +87,7 @@ public String getWelcomeFile(String pathInContext)
         for (String s : _welcomes)
         {
             String welcomeInContext = URIUtil.addPaths(pathInContext, s);
-            Resource welcome = newResource(welcomeInContext);
+            Resource welcome = _baseResource.resolve(welcomeInContext);
```

Comment:
I think `_baseResource.resolve()` should be moved to a new `protected resolve(String)` method akin to DefaultServlet.

Reply:
Good point about ee9.